pkgs/development/python-modules/pygame_sdl2/default.nix

```diff
@@ -4,12 +4,12 @@
 buildPythonPackage rec {
   pname = "pygame_sdl2";
   version = "2.1.0";
-  renpy_version = "7.2.0";
+  renpy_version = "8.0.0";
   name = "${pname}-${version}-${renpy_version}";
```

Comment:
```suggestion
```
Can we get this from renpy? Something like ``${lib.versions.majorMinor renpy.version}.${lib.versions.patch renpy.version}

Reply:
Done.